src/wagtailtrans/middleware.py

```diff
@@ -38,6 +38,6 @@ def process_request(self, request):
         request.LANGUAGE_CODE = active_language
 
     def process_response(self, request, response):
-        if 'Content-Language' not in response:
+        if 'Content-Language' not in response and hasattr(request, 'LANGUAGE_CODE'):
```

Comment:
Looks valid, however this means that there is a case where the request has no language code.
as `request.LANGUAGE_CODE` is widely used isn't it a better solution to set the language code again in the the response-request? 
for example:
```python
if not hasattr(request, 'LANGUAGE_CODE'):
    request.LANGUAGE_CODE = self.get_active_language(request) # method which determines the language based on path or browser preference (just like process request)
```